spec/support/time.cr

```diff
@@ -101,7 +106,8 @@ end
           ],
         )
         if LibC.AdjustTokenPrivileges(adjust_token, 0, pointerof(new_privileges), 0, nil, nil) != 0
-          return true
+          # ERROR_NOT_ALL_ASSIGNED means the privilege was not adjusted even though the return value is nonzero
```

Comment:
**question:** What is `ERROR_NOT_ALL_ASSIGNED` and where is it used?

Reply:
From the `AdjustTokenPrivileges` docs:

> The token does not have one or more of the privileges specified in the NewState parameter. The function may succeed with this error value even if no privileges were adjusted.

Reply:
Ah, so that's the only other value `WinError.value` can take and `.error_success?` effectively tests for the opposite.
The comment is far from clear to me.
Maybe enhance it a bit like this?
```suggestion
          # Check the error values is actually ERROR_SUCCESS.
          # ERROR_NOT_ALL_ASSIGNED would mean the privilege was not adjusted even though the return value is nonzero
```
I think it would be fine to drop the comment entirely. That might be less confusing than with it.